From 195a0c6dc1d713ef15b371f0db063c0607f58146 Mon Sep 17 00:00:00 2001
From: Seán de Búrca <leftmostcat@gmail.com>
Date: Mon, 26 Aug 2013 21:57:28 +0000
Subject: Fix autogen and configure

aclocal needs an explicit macro dir and side-by-side use of IT_PROG_INTLTOOL and AM_GNU_GETTEXT is not supported.

https://bugzilla.gnome.org/show_bug.cgi?id=672001
---
diff --git a/Makefile.am b/Makefile.am
index c176536..170b17c 100644
--- Makefile.am
+++ Makefile.am
@@ -1,3 +1,5 @@
 SUBDIRS = data help po src
 
+ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
+
 EXTRA_DIST = COPYING.docs
diff --git a/configure.ac b/configure.ac
index 112c3ba..9abced5 100644
--- configure.ac
+++ configure.ac
@@ -21,9 +21,6 @@ PKG_PROG_PKG_CONFIG([0.22])
 
 AC_PATH_PROG(GLIB_GENMARSHAL, [glib-genmarshal])
 
-AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.17])
-
 AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_CXX
--
cgit v0.9.2
